server/pbenchinacan/etc/rsyslog.d/95-output-viaq-forwarding.conf

```diff
@@ -12,48 +12,48 @@ if strlen($!msg) < 1 then {
     set $!msg = $msg;
 }
 
-# Forward messages via TCP, using JSON embedded in message (RFC 5424), with a
-# disk-assisted queue.
-action(
-    name="fwd-viaq"
-    type="omfwd"
-    Target="10.1.170.92"
-    Port="514"
-    Protocol="tcp"
-    Template="ViaQ_SyslogProtocol23Format"
-    # Rebind every thousand messages sent
-    RebindInterval="1000"
-
-    # In-memory linked-list queue, but because filename is defined it is disk-assisted
-    # See http://www.rsyslog.com/doc/v8-stable/concepts/queues.html?highlight=disk%20assisted
-    queue.type="linkedlist"
-    # Directory where the queue files on disk will be stored
-    queue.spoolDirectory="/var/lib/rsyslog/forwarder.q"
-    # Prefix of the name the queue files on disk
-    queue.filename="daq"
-    # Only store up to a maximum size of logs on disk
-    queue.maxDiskSpace="64m"
-    # Use 100 MB queue files
-    queue.maxFileSize="100m"
-    # Update disk queue every "so many" messages
-    queue.checkpointInterval="1024"
-    # Fsync when a check point occurs
-    queue.syncQueueFiles="on"
-    # Save messages to disk on shutdown
-    queue.saveOnShutdown="on"
-    # Allow up to 6 threads processing items in the queue
-    queue.workerThreads="6"
-
-    # Keep a "small" queue since journal persistence is assumed
-    queue.size="8192"
-
-    # Be sure we don't dequeue too many log messages at one time in case of failure
-    queue.DequeueBatchSize="2048"
-    # We have a minimum size to avoid making too many bulk API calls
-    queue.minDequeueBatchSize="128"
-    # Indexing no less than once a second to avoid pounding the target
-    queue.minDequeueBatchSize.timeout="1000"
-
-    # Infinite retries if host is down
-    action.resumeRetryCount="-1"
-)
+## Forward messages via TCP, using JSON embedded in message (RFC 5424), with a
```

Comment:
The support for `rsyslog` in the functional test deployment is...let us say, "evolving".  Apparently, it needs to be intimately related to the configuration on the host where the container is running.  (E.g., this file should probably be copied from the _host's_ `/etc/rsyslog.d` rather than from our Git repo checkout...but we cannot simply copy _all_ the files from that directory -- the tags file needs to be tweaked.)  So, until we get those details nailed down, Peter suggested that I comment out the `action()`, especially since its `Target` probably isn't the right value.